xml/Microsoft.Extensions.Logging/ILogger.xml

```diff
@@ -141,7 +141,7 @@
         <param name="eventId">Id of the event.</param>
         <param name="state">The entry to be written. Can also be an object.</param>
         <param name="exception">The exception related to this entry.</param>
-        <param name="formatter">Function to create a <see cref="T:System.String" /> message of the <paramref name="state" /> and <paramref name="exception" />.</param>
+        <param name="formatter">Function to create a <see cref="T:System.String" /> message of the <paramref name="state" /> and <paramref name="exception" />. **Note: When using extension methods such as <see cref="M:Microsoft.Extensions.Logging.LoggerExtensions.Log(Microsoft.Extensions.Logging.ILogger,Microsoft.Extensions.Logging.LogLevel,Microsoft.Extensions.Logging.EventId,System.Exception,System.String,System.Object[])"/> and <see cref="M:Microsoft.Extensions.Logging.LoggerExtensions.LogError(Microsoft.Extensions.Logging.ILogger,System.Exception,System.String,System.Object[])"/>, the function passed here will not add the <paramref name="exception"/> parameter to the log message. When using these overloads, implementations should explicitly append the <paramref name="exception"/> to the log message.**</param>
```

Comment:
>. **Note: When using extension methods such as <see cref="M:Microsoft.Extensions.Logging.LoggerExtensions.Log(Microsoft.Extensions.Logging.ILogger,Microsoft.Extensions.Logging.LogLevel,Microsoft.Extensions.Logging.EventId,System.Exception,System.String,System.Object[])"/> and <see cref="M:Microsoft.Extensions.Logging.LoggerExtensions.LogError(Microsoft.Extensions.Logging.ILogger,System.Exception,System.String,System.Object[])"/>, the function passed here will not add the <paramref name="exception"/> parameter to the log message. When using these overloads, implementations should explicitly append the <paramref name="exception"/> to the log message.**</param> [](http://example.com/codeflow?start=158&length=666)

Would be better if we move this text to the remarks element instead.

Reply:
Moved the note to the remarks

Reply:
You missed the `=` in the part `<paramref name "formatter">`

Reply:
Fixed